src/panels/config/integrations/ha-config-integrations.ts

```diff
@@ -170,11 +170,12 @@ class HaConfigIntegrations extends SubscribeMixin(LitElement) {
         filter
       );
       const ignored: ConfigEntryExtended[] = [];
-      filteredConfigEnties.forEach((item, index) => {
-        if (item.source === "ignore") {
-          ignored.push(filteredConfigEnties.splice(index, 1)[0]);
+      let i = filteredConfigEnties.length;
+      while (i--) {
```

Comment:
Is this better than a for loop? I feel like they are the same but a for loop is more readable as its more common